dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/datavalueset/DefaultDataValueSetService.java

```diff
@@ -755,238 +740,53 @@ public ImportSummary saveDataValueSetPdf( InputStream in, ImportOptions importOp
     private ImportSummary saveDataValueSet( ImportOptions importOptions, JobConfiguration id,
         DataValueSet dataValueSet )
     {
-        importOptions = ObjectUtils.firstNonNull( importOptions, ImportOptions.getDefaultImportOptions() );
+        final ImportContext context = createDataValueSetImportContext( importOptions, dataValueSet );
+        logDataValueSetImportContextInfo( context );
```

Comment:
Note that all logging that previously was in-between some of the preparation lines is now done after the preparation is complete (the `ImportContext` is created). This does not change anything output wise but order of operations has changes slightly.